server/pension/management/commands/import-csv.py

```diff
@@ -0,0 +1,52 @@
+
+from django.core.management import BaseCommand
+from pension.models import Quarter, ManagingBody, Fund, Instrument, Holding
+import csv, codecs
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        print("Hello Command")
+        csv_filename="C:/Users/user/Documents/PycharmProjects/open-pension/dummy_data.csv"
+
+        # TODO Handle Hebrew Characters
+        print("opening csv file")
+        data_reader = csv.reader(open(csv_filename, 'r', encoding='utf-8'), delimiter=',', quotechar='"')
+        print("opened csv file")
+
+        for row in data_reader:
+            if str(row[0]).find('managing_body') == -1:
+
+                # mbody = ManagingBody.objects.get(pk=row[0])
+                if not(ManagingBody.objects.filter(label=row[0]).exists()):
+                    print("saving Managing Body")
+                    managing_body = ManagingBody(label=row[0])
```

Comment:
we should use [get_or_create](https://docs.djangoproject.com/en/1.9/ref/models/querysets/#get-or-create) on these cases.